flink-contrib/docker-flink/docker-entrypoint.sh

```diff
@@ -28,6 +28,9 @@ if [ "$1" == "jobmanager" ]; then
 
     echo "config file: " && grep '^[^\n#]' $FLINK_HOME/conf/flink-conf.yaml
     $FLINK_HOME/bin/jobmanager.sh start cluster
+
+  # prevent script to exit
+  tail -f /dev/null
```

Comment:
I think the proper way to fix this, would be to call a non-daemonized startup script.